bindings/ocaml/SharedDefs.ml

```diff
@@ -41,7 +48,46 @@ end
 module Hacl_Hash = Hacl_Hash_bindings.Bindings(Hacl_Hash_stubs)
 module Hacl_Spec = Hacl_Spec_bindings.Bindings(Hacl_Spec_stubs)
 
-let max_size_t = Z.of_string "4294967296"
+let pow2 n = Z.(pow ~$2) n
```

Comment:
Never seen that `~$2` syntax, what does it mean?

Reply:
It's a Zarith prefix operator for converting from int.

Reply:
Oh you taught me something here, from now on I'll stop struggling with `Z.of_int` :)